q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'fit_prior': False}]]]]
```

Comment:
Is this `fit_prior` change something we should notify users about in the release notes/announcement in addition to the change in default confidence? If so, can you provide some text describing what the change is and how it affects users? Then I can add the text to our release notes draft.